lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -312,7 +312,8 @@ def __init__(self, bbox1, bbox2, loc1, loc2=None, **kwargs):
             raise ValueError("transform should not be set")
 
         kwargs["transform"] = IdentityTransform()
-        Patch.__init__(self, fill=False, **kwargs)
+        fill = 'fc' in kwargs
```

Comment:
See https://github.com/matplotlib/matplotlib/blob/e69ea36995a76c37b928f9cab9af1161f8d36e05/lib/matplotlib/axes/_axes.py#L1406 for an example.

You can use `normalize_kwargs` to sort out `fc` vs `facecolor`, etc .  You will have to do the color / facecolor / edgecolor normalization by hand I think.